src/main/app/src/assets/i18n/nl.json

```diff
@@ -339,27 +339,27 @@
   "actie.admin": "Beheer-instellingen",
   "actie.alles.deselecteren": "Alles deselecteren",
   "actie.alles.selecteren": "Alles selecteren",
-  "actie.annuleren": "Annuleren",
-  "actie.bagobject.toevoegen": "BAG object toevoegen",
-  "actie.besluit.vastleggen": "Besluit vastleggen",
-  "actie.betrokkene.toevoegen": "Betrokkene toevoegen",
-  "actie.betrokkene.toevoegen.persoon": "Persoon als betrokkene toevoegen",
-  "actie.betrokkene.toevoegen.bedrijf": "Bedrijf als betrokkene toevoegen",
-  "actie.betrokkene.verwijderen": "Betrokkene verwijderen",
-  "actie.bewerken": "Bewerken",
-  "actie.card.toevoegen": "Dashboardcard toevoegen",
-  "actie.card.verwijderen": "Dashboardcard verwijderen",
-  "actie.dashboard.wijzigen": "Uw dashboard aanpassen...",
-  "actie.deselecteren": "Deselecteren",
-  "actie.document.toevoegen": "Document toevoegen",
-  "actie.document.toevoegen.nogmaals": "Nog een document toevoegen",
-  "actie.document.maken": "Document maken",
-  "actie.document.ontkoppelen": "Document ontkoppelen",
-  "actie.document.verplaatsen": "Document verplaatsen",
-  "actie.document.verplaatsen.disabled": "Ga naar de zaak waar dit document naar verplaatst moet worden...",
-  "actie.document.verwijderen": "Document verwijderen",
-  "actie.document.verwijderen.reden": "Reden waarom document wordt verwijderd",
-  "actie.document.bekijken": "Document bekijken",
+  "actie.annuleren" : "Annuleren",
+  "actie.bagobject.toevoegen" : "BAG object toevoegen",
+  "actie.besluit.vastleggen" : "Besluit vastleggen",
+  "actie.betrokkene.toevoegen" : "Betrokkene toevoegen",
+  "actie.betrokkene.toevoegen.persoon" : "Persoon als betrokkene toevoegen",
+  "actie.betrokkene.toevoegen.bedrijf" : "Bedrijf als betrokkene toevoegen",
+  "actie.betrokkene.verwijderen" : "Betrokkene verwijderen",
+  "actie.bewerken" : "Bewerken",
+  "actie.card.toevoegen" : "Dashboardcard toevoegen",
+  "actie.card.verwijderen" : "Dashboardcard verwijderen",
+  "actie.dashboard.wijzigen" : "Pas uw dashboard aan...",
```

Comment:
Waarom staan hier 3 punten?

Reply:
Omdat het geen directe actie is maar een actie die andere acties mogelijk maakt. Eigenlijk hoor je dat ook te gebruiken als je bijvoorbeeld een dialog (of panel) opent.